### PR TITLE
Allow some config to be set in ENV

### PR DESCRIPTION
**Pull request by dekz**
#14 
```
RPC_URL=http://localhost:8545 NETWORK_ID=50 HTTP_PORT=4000 yarn start:ts
```

Exposing some common config items to be set in ENV vars.

Thoughts on `WHITELISTED_TOKENS` so it can quickly be spun up on multiple networks without rebuilding? Might require a bit of parsing to sort out '*' versus an csv tokens.

### CODE REVIEW

**Review by LogvinovLeon — APPROVED** [review]
LGTM, but let's see what @fabioberger has to say

File: ts/src/config.ts [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+export const HTTP_PORT = process.env.HTTP_PORT || 3000;
 // A time window after which the order is considered permanently expired
 export const ORDER_SHADOWING_MARGIN_MS = 100 * 1000; // tslint:disable-line custom-no-magic-numbers
 // Frequency of checks for permanently expired orders
 export const PERMANENT_CLEANUP_INTERVAL_MS = 10 * 1000; // tslint:disable-line custom-no-magic-numbers
 // Max number of entities per page
 export const MAX_PER_PAGE = 100;
 // Default network id to use when not specified
-export const NETWORK_ID = 42;
+export const NETWORK_ID = !_.isUndefined(process.env.NETWORK_ID) ? parseInt(process.env.NETWORK_ID, 10) : 42;
```
Comment (fabioberger) [inline ts/src/config.ts]:
```suggestion
export const NETWORK_ID = !_.isUndefined(process.env.NETWORK_ID) ? _.parseInt(process.env.NETWORK_ID) : 42;
```

**Comment by fabioberger** [comment]
@dekz is this a need users have expressed? Not sure how common of an ask this is.

**Comment by laurengarcia** [comment]
I would like this very much to make spinning up parallel versions (dev,test,prod) on the same branch easy. Was actually fretting about this earlier and just now came across this PR, happily.

**Comment by laurengarcia** [comment]
Might want to also do: `FEE_RECIPIENT` and the `MAKER_FEE...` and `TAKER_FEE...` as well.

**Review by fabioberger — APPROVED** [review]

**Pull request merged**